extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-input.js

```diff
@@ -32,10 +32,14 @@ describes.realWin('amp-recaptcha-input', {
     toggleExperiment(win, 'amp-recaptcha-input', true);
   });
 
-  function getRecaptchaInput(skipSiteKey, skipAction) {
+  function getRecaptchaInput(skipName, skipSiteKey, skipAction) {
```

Comment:
nit: flag booleans in test methods can make the tests difficult to grok sometimes. Consider using a config object with these as properties instead. That way in the tests you can label the keys at the call sites

```js
// This can be easier to understand
getRecaptchaInput({skipAction: true});

// This can be harder to understand
getRecaptchaInput(undefined, undefined, true);

// This is better but not 💯
getRecaptchaInput(/*opt_skipName*/ undefined, /*opt_skipSiteKey*/ undefined, /*opt_skipAction*/ true);
```

Reply:
Yeah it got kinda out of hand. Thanks! Will do this.